egui/src/widgets/plot/items/mod.rs

```diff
@@ -61,7 +61,13 @@ pub(super) trait PlotItem {
         }
     }
 
-    fn on_hover(&self, elem: ClosestElem, shapes: &mut Vec<Shape>, plot: &PlotConfig<'_>) {
+    fn on_hover(
+        &self,
+        elem: ClosestElem,
+        shapes: &mut Vec<Shape>,
+        plot: &PlotConfig<'_>,
+        custom_label_func: &CustomLabelFunc,
```

Comment:
We allow the user to pass a custom function that formats the hover string based on input value.